usr/bin/nxdk-c++

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+clang -target i386-pc-win32 -march=pentium3 \
+-ffreestanding -nostdlib -fno-builtin -fno-exceptions \
+-I${NXDK_DIR}/lib/libcxx/include \
```

Comment:
A lot of redundancy with C version, but this must come first. Should probably include flags from some Makefile_rules file?